lib/moduleConfig.js

```diff
@@ -1,19 +1,24 @@
 "use strict";
 delete require.cache[__filename]; //do not cache in require cache
 delete require.cache[module.parent.filename]; //do not cache in require cache
-var cache = require("./cache"); //use requires caching to have a singleton
-var getConfigPath = require("./getConfigPath"); //use requires caching to have a singleton
-var path = require("path");
+const cache = require("./cache"); //use requires caching to have a singleton
+const getConfigPath = require("./getConfigPath"); //use requires caching to have a singleton
+const path = require("path");
 
-module.exports = function moduleConfig(paths, loadPathFunction) {
-	var startDir = path.dirname(module.parent.parent.filename);
-	var configPath;
-	var pathsId = paths.join(",");
+module.exports = function moduleConfig(startDir, paths, loadPathFunction) {
+	if (Array.isArray(startDir)) {
+		loadPathFunction = paths;
+		paths = startDir;
+		startDir = path.dirname(module.parent.parent.filename);
+	}
```

Comment:
This is the change that will allow to use this module with Honeycomb enabled.

Other changes in this PR are just formatting/refactoring.